src/server/handlers/openapi/mojaloop/index.ts

```diff
@@ -41,7 +42,7 @@ export const apiHandlers = {
   postAuthorizations: MojaloopAuthorizations.post,
   postConsents: MojaloopConsents.post,
   putConsentsById: MojaloopConsentsById.put,
-  deleteConsentsById: MojaloopConsentsById.remove,
```

Comment:
I'm guessing the patch is due to fact we don't want to remove consents but rather change their status to 'revoked'?

Reply:
yup - but the app is free to do differently I believe